src/Simplex/Messaging/Agent.hs

```diff
@@ -164,10 +164,11 @@ processCommand c@AgentClient {sndQ} (corrId, connAlias, cmd) =
         _ -> throwError PROHIBITED -- NOT_READY ?
       where
         sendMsg sq = do
-          withStore $ \st -> createSndMsg st connAlias msgBody
+          senderTs <- liftIO getCurrentTime
```

Comment:
this timestamp should be the same as timestamp used in `mkAgentMessage` - it should be passed there, so they are the same

Reply:
and it should also be the same as internalTs I think for sent messages